components/brave_rewards/resources/img/ads_disabled.svg

```diff
@@ -0,0 +1 @@
+<svg width="106" height="80" xmlns="http://www.w3.org/2000/svg"><g fill="none" fill-rule="evenodd"><path d="M77.882 21.825V2.67c0-.998-.542-1.888-1.432-2.344A2.678 2.678 0 0 0 73.696.5C54.654 13.668 43.723 15.361 36.327 15.361H15.182C6.81 15.36 0 22.17 0 30.547v5.64c0 8.375 6.81 15.186 15.182 15.186h10.15L36.349 78.08a2.655 2.655 0 0 0 2.451 1.65c.346 0 .673-.067 1.019-.197 1.367-.564 1.995-2.104 1.453-3.47l-10.193-24.69h5.227c25.765 0 36.305 13.646 36.783 14.276a2.646 2.646 0 0 0 2.95.954 2.658 2.658 0 0 0 1.82-2.516V44.952c5.271-1.216 9.197-5.923 9.197-11.564.043-5.64-3.905-10.37-9.174-11.563zM5.314 36.187v-5.64c0-5.446 4.424-9.872 9.868-9.872h9.281v25.384h-9.281c-5.444 0-9.868-4.426-9.868-9.872zm67.254 21.5c-5.899-4.86-17.48-11.628-36.262-11.628h-6.55V20.675h6.55c7.591 0 18.477-1.562 36.262-13.017v50.029zm5.314-18.333V27.401a6.535 6.535 0 0 1 3.882 5.987c.021 2.647-1.583 4.947-3.882 5.966z" fill="#CEB4E1"/><path d="M103.25 30H92.539c-1.521 0-2.75 1.118-2.75 2.5s1.229 2.5 2.75 2.5h10.71c1.521 0 2.751-1.118 2.751-2.5 0-1.402-1.23-2.5-2.75-2.5zM88.712 24.651a2.6 2.6 0 0 0 2.407 1.599c.317 0 .635-.061.951-.164l9.797-3.732c1.33-.513 1.984-1.969 1.456-3.26-.528-1.293-2.026-1.93-3.357-1.416L90.17 21.41a2.493 2.493 0 0 0-1.458 3.24zM101.868 43.91l-9.79-3.732c-1.328-.513-2.827.123-3.354 1.415-.528 1.291.127 2.748 1.456 3.26l9.788 3.734c.318.122.634.163.95.163a2.596 2.596 0 0 0 2.405-1.6c.528-1.271-.126-2.727-1.455-3.24z" fill="#F3CD72"/><text font-family="Poppins-SemiBold, Poppins" font-size="12" font-weight="500" fill="#9752CB"><tspan x="41" y="32">Ads</tspan></text></g></svg>
```

Comment:
where do we use this?

Reply:
in Android specific HTML page

Reply:
Seems to be not used anymore. Will be deleted.

Reply:
Removed